### PR TITLE
feat: Add postVerifyPhoneCode to AccountAPI

### PR DESCRIPTION
**Pull request by bardiarastin98**

### CODE REVIEW

File: packages/api-client/src/account/AccountAPI.ts [hunk 1]
```diff
@@ -130,6 +130,34 @@ export class AccountAPI {
     await this.client.sendJSON(config);
   }
 
+  /**
+   * Verify phone sms code
```
Comment (ffflorian) [inline packages/api-client/src/account/AccountAPI.ts:134]:
```suggestion
   * Activate (i.e. confirm) a phone number
```

File: packages/api-client/src/account/AccountAPI.ts [hunk 2]
```diff
@@ -130,6 +130,34 @@ export class AccountAPI {
...
+    };
+    try {
+      const response = await this.client.sendJSON(config);
+      if (response.status === 200) {
+        return true;
+      }
+      if (response.status === 404) {
+        return false;
+      }
+      return false;
```
Comment (ffflorian) [inline packages/api-client/src/account/AccountAPI.ts:155]:
```suggestion
      return response.status === 200;
```

File: packages/api-client/src/account/AccountAPI.ts [hunk 3]
```diff
@@ -130,6 +130,34 @@ export class AccountAPI {
...
+    } catch (error) {
+      throw error;
```
Comment (ffflorian) [inline packages/api-client/src/account/AccountAPI.ts:157]: try/catch is not needed if you are not doing anything with it.

Comment (bardiarastin98) [reply packages/api-client/src/account/AccountAPI.ts]: done in https://github.com/wireapp/wire-web-packages/pull/3050/commits/ea1aa0c2f970faf53850df0392e6a1ea566e6b42

**Pull request closed without merging**